task/buildah/0.1/buildah.yaml

```diff
@@ -107,29 +107,36 @@ spec:
       sed -i 's/^short-name-mode = .*/short-name-mode = "disabled"/' /etc/containers/registries.conf
 
       if [ "${HERMETIC}" == "true" ]; then
-        NETWORK="--network none"
+        BUILDAH_ARGS="--pull=never"
+        UNSHARE_ARGS="--net"
+        for image in $(grep -i '^FROM' "$dockerfile_path" | awk '{print $2}'); do
```

Comment:
Not too long ago we had a bug filed in dockerfile-parse for `FROM --platform=...`: https://github.com/containerbuildsystem/dockerfile-parse/issues/124

Should we try to deal with those cases?